src/action.c

```diff
@@ -460,19 +505,9 @@ actions_run(struct view *activator, struct server *server,
 			break;
 		case ACTION_TYPE_SEND_TO_DESKTOP:
 			if (view) {
-				struct action_arg *arg;
-				char *target_name = NULL;
-				struct workspace *target;
-				bool follow = true;
-
-				wl_list_for_each(arg, &action->args, link) {
-					if (!strcmp("to", arg->key)) {
-						target_name = action_str_from_arg(arg);
-					} else if (!strcmp("follow", arg->key)) {
-						follow = action_bool_from_arg(arg);
-					}
-				}
-				target = workspaces_find(view->workspace, target_name);
+				char *to = str_from_key(action, "to", "");
```

Comment:
Instead of `""` I guess we could also use `NULL` here. I am not sure how `workspaces_find()` handles it though.

Reply:
Done. workspaces_find() is good with it.